package.json

```diff
@@ -1,9 +1,13 @@
 {
   "name": "rendezvous",
-  "version": "0.0.1",
+  "version": "0.0.2",
   "description": "Meet your contract's vulnerabilities head-on.",
   "main": "app.js",
+  "bin": {
+    "rv": "./dist/app.js"
+  },
   "scripts": {
+    "build": "npx tsc --project tsconfig.json && chmod +x ./dist/app.js",
```

Comment:
```suggestion
    "prepare": "npx tsc --project tsconfig.json",
```

Reply:
What are your thoughts on using `prepare` in package.json @hugocaillard? This worked on Debian, and run automatically after `npm install` so I didn't have to perform any extra build step.

---

(Note the `prepare` step below.)

```
$ npm install

> rendezvous@0.0.2 prepare
> npx tsc --project tsconfig.json


up to date, audited 354 packages in 2s
```

```
$ npm install --global .

added 1 package in 3s
```

```
$ which rv
/home/nikos/.nvm/versions/node/v20.18.0/bin/rv
```

```
$ cd ~
nikos@debian-linux:~
$ rv

No path to Clarinet project provided. Supply it immediately or ...

  rv v0.0.2

  Usage: ...
```

And we should also add the `uninstall` step too in the README:

```
nikos@debian-linux:~
$ npm uninstall --global rendezvous

removed 1 package in 182ms
```

Reply:
This approach does not work on Linux. The `npm install --global .` after all the previous steps were done:
```
$ npm install

...

> rendezvous@0.0.2 prepare
> npx tsc --project tsconfig.json


added 353 packages, and audited 354 packages in 5s

...

found 0 vulnerabilities
```
```
$ npm install --global .
npm error code 1
npm error path /home/radone/rendezvous
npm error command failed
npm error command sh -c npx tsc --project tsconfig.json
npm error
npm error                 This is not the tsc command you are looking for
npm error
npm error
npm error To get access to the TypeScript compiler, tsc, from the command line either:
npm error
npm error - Use npm install typescript to first add TypeScript to your project before using npx
npm error - Use yarn to avoid accidentally running code from un-installed packages
npm error npm warn exec The following package was not found and will be installed: tsc@2.0.4
npm error A complete log of this run can be found in: /home/radone/.npm/_logs/2024-12-21T15_13_37_344Z-debug-0.log
```

Reply:
The tests above were done on a clean Debian (my tests), and the failure (in your output) is due to the missing TypeScript compiler. Interesting. 🤔

Reply:
I've added https://github.com/stacks-network/rendezvous/issues/75 to help us iterate faster.